### PR TITLE
[Store Customization] Update the default content in patterns

### LINKED ISSUE #11875: Store Customization > Ensure the default content in patterns is updated with new copy provided by design team
Ensure all content in patterns used in the Assembler is updated based on the new copy to be provided by @jarekmorawski. This content will be used as a fallback for all relevant patterns both on core and also on WooExpress whenever AI is unavailable. 

Screenshots from Figma:

## Intro Patterns:

Intro patterns

## Featured selling patterns:

Featured selling patterns

## About patterns:

About patterns

## Reviews patterns:

Reviews patterns

## Social media patterns:

Social media patterns

### PR DESCRIPTION
**Pull request by albarin**
## What

This PR updates the copy of some patterns and makes some small pattern tweaks and improvements.

Fixes https://github.com/woocommerce/woocommerce-blocks/issues/11875

## Why

To ensure it matches with the new copy provided by design.

## Testing Instructions

_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Insert [all the patterns](https://i.imgflip.com/87rb8e.jpg) from the next list:
- Intro patterns
  - Featured category cover image
  - Hero product split
  - Just arrived full hero
- Featured selling patterns:
  - Product Collection 4 Columns
  - Product Collection 5 Columns
  - Product Collection 3 Columns
  - Product Collection: Featured Products 5 Columns
  - Featured category triple
  - Large image product gallery
  - Product gallery
  - Hero product 3-split
  - Hero product chessboard
  - Featured category focus
  - Banner
- About patterns
  - Alternating image and text
  - Testimonials 3 columns
  - Testimonials single
- Review patterns
  - Testimonials 3 Columns
  - Testimonials single
- Social media patterns
  - Social: Follow us on social media
3. Check the copy is the same expected in the designs (see screenshots here: https://github.com/woocommerce/woocommerce-blocks/issues/11875).

* [ ] Do not include in the Testing Notes 
* [ ] Should be tested by the development team exclusively 

## Screenshots or screencast

| Before | After |
| ------ | ----- |
|        |       |

## WooCommerce Visibility

Required:

* [x] WooCommerce Core
* [ ] Feature plugin
* [ ] Experimental
* [ ] N/A

## Checklist

Required:

* [x] This PR has either a `[type]` label or a `[skip-changelog]` label.
* [ ] This PR is assigned to a milestone.

Conditional:

* [ ] This PR has a UI change and has been cross-browser tested at different viewport sizes on both the frontend and in the editor.
* [x] This PR has a changelog description (if `[skip-changelog]` label is not present).
* [ ] This PR adds/removes a feature flag & I've updated [this doc](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/docs/internal-developers/blocks/feature-flags-and-experimental-interfaces.md).
* [ ] This PR adds/removes an experimental interfaces, and I've updated [this doc](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/docs/internal-developers/blocks/feature-flags-and-experimental-interfaces.md).
* [ ] This PR has been accessibility tested.
* [ ] This PR has had any necessary documentation added/updated.

## Changelog

> Update patterns copy

### CODE REVIEW

**Comment by albarin** [comment]
Hey @jarekmorawski! I've added you as well as a reviewer of the copy updates in this PR.
Also, I have some comments and questions that came up while working on this, could you have a look? 🙏 
- **Just arrived full hero**: the image is reversed and the text is not in the middle anymore, should I update the code to look like the designs or only update the copy?
- **3-column products row**: in the designs the “Our newest arrivals” title is not present, should I remove it?
- **Product gallery**: in the designs the “Our newest arrivals” title is not present, should I remove it?
- **Hero product chessboard**: the design was changed to only have 2 sections on the bottom/left square, I’ve picked the 2 first sections copy.

Thanks! 🙌

File: src/Patterns/dictionary.json [hunk 1]
```diff
@@ -63,9 +63,15 @@
 		"content": {
 			"titles": [
 				{
-					"default": "Announcing our newest collection",
+					"default": "Black and white high-quality prints",
```
Comment (nefeline) [inline src/Patterns/dictionary.json:66]: I see this pattern has two different copies on Figma:

Screenshot 2023-11-30 at 17 34 24

Do we have a confirmation of what copy is the correct one?

File: src/Patterns/dictionary.json [hunk 2]
```diff
@@ -171,31 +177,37 @@
 		"content": {
 			"titles": [
 				{
-					"default": "Quality Materials",
+					"default": "Inspired living spaces",
```
Comment (nefeline) [inline src/Patterns/dictionary.json:180]: Our implementation for this pattern differs from the design, with two sections missing.

Design:

Screenshot 2023-11-30 at 17 48 48


Implemented:

Screenshot 2023-11-30 at 17 48 07

Do you happen to remember if there was any previous discussion that resulted in this change @albarin ?

**Review by nefeline — COMMENTED** [review]
Greak work here @albarin ! These are my test results:

- ✅ Banner (woocommerce-blocks/banner) OK
- ❓ Featured Category Focus (woocommerce-blocks/featured-category-focus) : it has two different copies for the same pattern: do we know which one is correct?
- ✅ Featured Category Triple (woocommerce-blocks/featured-category-triple) OK
- ✅  Hero Product 3 Split (woocommerce-blocks/hero-product-3-split) OK
- ❓  Hero Product Chessboard (woocommerce-blocks/hero-product-chessboard) copy OK. Design is different
- ✅  Hero Product Split (woocommerce-blocks/hero-product-split) OK
- ✅  Just Arrived Full Hero (woocommerce-blocks/just-arrived-full-hero) OK
- ✅  Product Collection 4 Columns (woocommerce-blocks/product-collection-4-columns) - renamed from 1:1 image 4-column products row OK
- ✅  Product Collection 5 Columns (woocommerce-blocks/product-collection-5-columns) OK
- ✅  Social: Follow us on social media (woocommerce-blocks/social-follow-us-in-social-media) OK
- ✅  Alternating Image and Text (woocommerce-blocks/alt-image-and-text) OK
- ✅ Testimonials 3 Columns (woocommerce-blocks/testimonials-3-columns) OK
- ✅ Testimonials Single (woocommerce-blocks/testimonials-single) OK
- ✅ Featured Category Cover Image (woocommerce-blocks/featured-category-cover-image) OK

### Default content for products

I noticed that for all patterns displaying real products, we didn't make any changes to the default placeholder content or image for the dummy products.

On a separate PR, what do you think about selecting 6 default images and content to assign to the dummy products? I would suggest using the ones selected for the **Product Gallery pattern**, since it already displays 6 products exactly, but could be any in the images used in patterns with products (e.g. Product Collection 5 Columns)

Screenshot 2023-11-30 at 18 27 43

Tagging @jarekmorawski here to confirm, since this is a design decision :) TLDR: we have multiple patterns featuring products, (e.g. Product Collection 5 Columns, Product Collection 4 Columns, Product Gallery, etc.), and each one of these patterns has different images/content assigned to them on Figma, but in reality, we only need six as those are real auto-generated products (created only if the store owner doesn't have any products added to their store yet) that will always be displayed whenever these patterns are inserted in the editor.

@albarin To implement this, we would have to update the ProductUpdater class by:

- Making sure the titles of those products match the designs (by updating them here: https://github.com/woocommerce/woocommerce-blocks/blob/fd7efc7bd0a9c931d9aa1011ba3308c675177fcf/src/Patterns/ProductUpdater.php#L155 )
- Making sure the images of those products match the designs  (by updating the logic in the ProductUpdater class: https://github.com/woocommerce/woocommerce-blocks/blob/53d3d9c0c19282d33c5bf292785070ef01721a98/src/Patterns/ProductUpdater.php#L298)

WDYT?

File: src/Patterns/dictionary.json (hunk 2 above)
Comment (nefeline) [reply src/Patterns/dictionary.json:180]: Oh, I see you already pinged Jarek about this https://github.com/woocommerce/woocommerce-blocks/pull/11997#issuecomment-1833844751, cool! Let's wait for his response! 🙌

**Review by nefeline — APPROVED** [review]
I'm going ahead and hitting the approve button as I can confirm the copy was correctly updated for all patterns: depending on the outcome of the discussion with Jarek, for potential changes in design on the **Hero Product Chessboard pattern** and the default placeholder content for dummy products, let's open separate PRs. 👍

File: src/Patterns/dictionary.json (hunk 1 above)
Comment (jarekmorawski) [reply src/Patterns/dictionary.json:66]: I don't know why we have two versions. I'll ask.

Comment (albarin) [reply src/Patterns/dictionary.json:66]: The current implementation of this pattern has the second picture, which is why I chose the `Black and white high-quality prints` copy. Not sure about the two versions though.

File: src/Patterns/dictionary.json (hunk 2 above)
Comment (albarin) [reply src/Patterns/dictionary.json:180]: > Do you happen to remember if there was any previous discussion that resulted in this change @albarin?

Yes, this was discussed with Veronica and Tom in this conversation p1698926001899609/1698918388.375659-slack-C053716F2H2

**Comment by albarin** [comment]
Thanks for your review @nefeline, I'm merging this PR.
I'll work on updating the image in a separate PR and let's wait for more feedback on the chessboard pattern.

**Comment by jarekmorawski** [comment]
> the image is reversed and the text is not in the middle anymore, should I update the code to look like the designs or only update the copy?

No, it should match the design. Otherwise, the pattern becomes more tricky to work with regarding imagery and layout.

> in the designs the “Our newest arrivals” title is not present, should I remove it?

> in the designs the “Our newest arrivals” title is not present, should I remove it?

Yes, please! Note that all titles should be aligned to the left, like in the designs. FlPl7fHaGv9mpu8Nfl0joH-fi-158_32452

> the design was changed to only have 2 sections on the bottom/left square, I’ve picked the 2 first sections copy.

Not sure why it was removed, but if that was the decision, I'll update the Figma file. As for the copy, I'd swap it for `Quality Materials` and `Unique Design`. FlPl7fHaGv9mpu8Nfl0joH-fi-154_27382

**Comment by albarin** [comment]
@jarekmorawski @nefeline This is the follow-up PR solving the latest Jarek's feeback.

**Pull request merged**